pkgs/games/teeworlds/default.nix

```diff
@@ -21,35 +21,12 @@ stdenv.mkDerivation rec {
                 '#define DATA_DIR "${placeholder "out"}/share/teeworlds/data"'
   '';
 
-  nativeBuildInputs = [ bam pkgconfig ];
+  nativeBuildInputs = [ cmake pkgconfig ];
 
-  configurePhase = ''
-    runHook preConfigure
-    bam config
-    runHook postConfigure
-  '';
-
-  buildPhase = ''
-    runHook preBuild
-    bam conf=release
-    runHook postBuild
-  '';
-
-  installPhase = ''
-    mkdir -p $out/bin $out/share/teeworlds
-    cp build/x86_64/release/teeworlds{,_srv} $out/bin
-    cp -r build/x86_64/release/data $out/share/teeworlds
-  '';
-
-  buildInputs = [
+    buildInputs = [
```

Comment:
```suggestion
  buildInputs = [
```

Reply:
Haha, d'oh! But thank you for catching and bringing it to my attention.  Fixup'd it into the main commit, should be good now :).